client/my-sites/theme/main.jsx

```diff
@@ -577,6 +595,13 @@ const ThemeSheetWithOptions = ( props ) => {
 	);
 };
 
+const mapDispatchToProps = ( dispatch, ownProps ) => ( {
+	recordThemeSheetButtonClick: ( context ) => dispatch( recordTracksEvent( 'calypso_theme_sheet_button_click', {
+		theme_name: ownProps.id,
+		button_context: context
+	} ) )
+} );
```

Comment:
I guess technically there is nothing wrong to do it this way, just note that if we ever decided to use `recordTracksEvent` somewhere else in this component it couldn't be used right away but would need to get its own mapping here